connector.go

```diff
@@ -77,6 +79,54 @@ type EntityInfo struct {
 	TTL *time.Duration
 }
 
+// StringSet is a set of strings.
+type StringSet map[string]struct{}
+
+// ScopeMetadata is metadata about a scope. (JSON tags to support MD setting CLI tools.) The scope
+// may be qualified by a prefix, as in "production.vsoffers".
+type ScopeMetadata struct {
+	Entity      `dosa:"primaryKey=(Name)" json:"-"`
+	Name        string     `json:"name"`
+	Owner       string     `json:"owner"` // Owning group name (or the same as Creator)
+	Type        int32      `json:"type"`  // Production, Staging, or Development
+	Flags       int64      `json:"flags"`
```

Comment:
what's flags?

Reply:
Initially I was going to add some parameters here but I changed the code and this is no longer needed.

I think in the future this will be useful, so my feeling is to leave it in. But I can remove it.